ubidots/ubidots_in.js

```diff
@@ -90,6 +89,31 @@ module.exports = function (RED) {
         }
       });
     });
+
+    function defineOutputObject(topic, message) {
+      let finalObject = {};
+      let variable = topic;
+      if (useCustomTopics) {
+        variable = topic.substring(14);
+        if (topic.substring(topic.length - 3) === "/lv") {
+          finalObject[variable] = { value: JSON.parse(message.toString()) };
+        } else {
+          finalObject[variable] = JSON.parse(message.toString());
+        }
+      } else {
+        if (topic.substring(topic.length - 3) === "/lv") {
+          variable = topic.slice(0, topic.length - 3);
+        }
+        let array = variable.split("/");
```

Comment:
What do you think about naming this variable a bit more explicit? I mean, the name could say what's the data that this array has.